_data/rowhitswami/rowhitswami.md

```diff
@@ -0,0 +1,7 @@
+---
+name: Rohit Swami # No longer than 18 characters
```

Comment:
Please do not include comments

Reply:
@Rishit-dagli It's not required to remove comments! Comments are fine in the markdown file.

Reply:
@Mohammed-Shoaib thank you, please approve the changes.